lib/chef/resource/route.rb

```diff
@@ -40,7 +41,6 @@ class Route < Chef::Resource
       property :hostname, [String, nil], desired_state: false
       property :domainname, [String, nil], desired_state: false
       property :domain, [String, nil], desired_state: false
-      property :metric, [Integer, nil], desired_state: false
```

Comment:
This looks wrong. metric is clearly an Integer.

Reply:
fixed :-)